Popular_Models_Guide/Llama2/trtllm_guide.md

```diff
@@ -336,6 +338,43 @@ curl -X POST localhost:8000/v2/models/ensemble/generate -d '{"text_input": "What
 > {"context_logits":0.0,...,"text_output":"What is ML?\nML is a branch of AI that allows computers to learn from data, identify patterns, and make predictions. It is a powerful tool that can be used in a variety of industries, including healthcare, finance, and transportation."}
 > ```
 
+### Running Gen-AI Perf
+Run the following command within the docker container:
```

Comment:
Let's briefly mention what genai perf is and link docs and repo: https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/client/src/c%2B%2B/perf_analyzer/genai-perf/README.html